doc/source/upgrading_guide_upstream.rst

```diff
@@ -0,0 +1,408 @@
+.. _upgrading_guide_upstream:
+
+================================================================================
+|Percona Server| In-Place Upgrading Guide from |MySQL|
+================================================================================
+
+In-place upgrades keep the existing data on the server. Generally speaking,
+in-place upgrades involve stopping the server, removing the installed version of
+|MySQL|, installing the new server, and starting it with the same data files.
+
+While in-place upgrades may not be suitable for high-complexity environments,
+they may be appropriate for many simpler scenarios.
+
+It is strongly recommended that you read the following guides to be aware of the
+incompatible changes that could cause the in-place upgrade to fail:
+
+* `Upgrading MySQL <http://dev.mysql.com/doc/refman/5.7/en/upgrading.html>`_
+* `Upgrading from MySQL 5.6 to 5.7 <http://dev.mysql.com/doc/refman/5.7/en/upgrading-from-previous-series.html>`_
+
+Use one of the following approaches to make the in-place upgrade to |Percona
+Server| 5.7 from |MySQL|.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _percona-server.in-place-upgrade.repository.upstream:
+
+In-place upgrade using |Percona| repositories
+================================================================================
+
+The recommended way of installing |Percona Server| is by using the |Percona|
+repositories.
+
+Instructions for your system about how to enable |Percona| repositories can be
+found in the section :ref:`installing_from_binaries`.
+
+The in-place upgrade using |Percona| repositories requires the following steps:
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _percona-server.in-place-upgrade.preliminary.upstream:
+
```

Comment:
One of the preliminary steps is to disable the upstream repo (oracle's mysql community) as it may cause conflicts when installing our server packages.